language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/components/tests/helpers/DummyLedgerClient.scala

```diff
@@ -137,6 +137,17 @@ class DummyLedgerClient(
           commands))
       Single.just(Empty.getDefaultInstance)
     }
+
+    // TODO Untested endpoint
```

Comment:
Same here.

Reply:
I'm trying to move the clients in parallel and this is a single file that everybody touches, I'll leave it alone for the time being but I will definitely apply this suggestion when I'm putting all the clients together in the final PR for this ticket. Thanks! :+1: